### PR TITLE
Support extensions on operations

### PR DESCRIPTION
**Pull request by isovector**
This PR adds a new `_operationExtensions` field to `Operation`, which correctly roundtrip.

### CODE REVIEW

**Review by fisx — APPROVED** [review]
thanks!

works in the [wire messenger](https://github.com/wireapp/wire-server/pull/2950), and the changes look good to me, but i'm not an active maintainer of this project, so i'd appreciate a second opinion.

(*are there* active maintainers?)

**Comment by fisx** [comment]
@isovector maybe you could add a haddocks description like in the other fields?

2023-01-11-222329_1600x900_scrot

**Review by swamp-agr — APPROVED** [review]

**Pull request merged**
